chandra_aca/tests/test_drift.py

```diff
@@ -0,0 +1,67 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from pathlib import Path
+
+import astropy.table as tbl
+import numpy as np
+import pytest
+
+from chandra_aca import drift
+
+
+def test_get_aca_offsets_legacy():
+    """
+    Legacy test that ACA offsets are reasonable, and regression test particular values
+    corresponding to cycle 17 zero-offset aimpoints used below for chip_x, chip_y inputs.
+
+    The output reference values here have been validated as being "reasonable" for the
+    given inputs.
+    """
+    offsets = drift.get_aca_offsets(
+        "ACIS-I", 3, 930.2, 1009.6, "2016:180:12:00:00", -15.0
+    )
+    assert np.allclose(offsets, (11.45, 2.34), atol=0.1, rtol=0)
+
+    offsets = drift.get_aca_offsets(
+        "ACIS-S", 7, 200.7, 476.9, "2016:180:12:00:00", -15.0
+    )
+    assert np.allclose(offsets, (12.98, 3.52), atol=0.1, rtol=0)
+
+    offsets = drift.get_aca_offsets("HRC-I", 0, 7591, 7936, "2016:180:12:00:00", -15.0)
+    assert np.allclose(offsets, (14.35, 0.45), atol=0.1, rtol=0)
+
+    offsets = drift.get_aca_offsets("HRC-S", 2, 2041, 9062, "2016:180:12:00:00", -15.0)
+    assert np.allclose(offsets, (16.89, 3.10), atol=0.1, rtol=0)
+
+
+# Generate test cases for get_aca_offsets(). This file was generated by the notebook
+# fit_aimpoint_drift-2022-11.ipynb in the aimpoint_mon repo.
+filename = Path(__file__).parent / "data" / "aimpoint_regression_data.ecsv"
```

Comment:
I'm a little confused about the new regression data setting things up to do per-detector tests, since the drift is not different per detector.  But if the point is just to have a broad range for end-to-end testing that the dynamic aimpoint calculations are working, this seems fine.

Reply:
That's true about the drift being independent of detector. This test data was driven by the easy availability of aimpoint offset regression data, while the legacy drift model outputs are not captured anywhere.

Another way to think of this is that is provides bonus testing of `get_aca_offsets` (which is explicitly detector-dependent) as well as the drift model. The original testing of `get_aca_offsets` (now `test_get_aca_offsets_legacy()`) was not very stringent.